common/namespace/replication_task_executor.go

```diff
@@ -130,8 +130,10 @@ func (h *namespaceReplicationTaskExecutorImpl) shouldProcessTask(ctx context.Con
 	})
 	switch err.(type) {
 	case nil:
-		if resp.Namespace.Info.Id != task.GetId() {
-			h.logger.Error("namespace replication encountered name / UUID collision during determination on whether to process task", tag.WorkflowNamespaceID(task.Info.GetName()))
+		if resp.Namespace.Info.GetId() != task.GetId() {
+			h.logger.Error(
+				"namespace replication encountered UUID collision processing NamespaceCreationReplicationTask",
+				tag.WorkflowNamespaceIDs(map[string]struct{}{resp.Namespace.Info.Id: {}, task.GetId(): {}, task.Info.GetId(): {}}))
```

Comment:
can we print `resp.Namespace.Info.GetId() and task.GetId()` explicitly in the error message? same applies to change at line 226 below.